packages/calling/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -195,11 +224,11 @@ describe('CallingClient Tests', () => {
     it('ContactCenter: verify empty invalid service domain', async () => {
       const serviceDataObj = {indicator: ServiceIndicator.CONTACT_CENTER, domain: ''};
 
-      expect(() => {
-        callingClient = createClient(webex, {serviceData: serviceDataObj});
-      }).toThrow(Error);
-      expect(callingClient).toBe(undefined);
-      expect(createRegistrationSpy).not.toHaveBeenCalled();
+      try {
+        callingClient = await createClient(webex, {serviceData: serviceDataObj});
+      } catch (e) {
+        expect(e.message).toMatch('Invalid service domain.');
```

Comment:
```suggestion
        expect(e.message).toEqual('Invalid service domain.');
```
Let's use toEqual() like all other tests which does full equality check.

Reply:
Done